src/compiler/diagnosticMessages.json

```diff
@@ -2824,5 +2827,9 @@
     "Unknown typing option '{0}'.": {
         "category": "Error",
         "code": 17010
+    },
+    "Too many JavaScript files in the project. Use an exact 'files' list, or use the 'exclude' setting in project configuration to limit included source folders. The likely folder to exclude is '{0}'. To disable the project size limit, set the 'disableSizeLimit' compiler option to 'true'": {

```

Comment:
consider updating the error message to:

`Too many JavaScript files in the project. Consider specifying the 'exclude' setting in project configuration to limit included source folders. 
The likely folder to exclude is '{0}'. 
To disable the project size limit, set the 'disableSizeLimit' compiler option to 'true'`